### ISSUE TITLE
HttpApp missing simplest startServer overload

### ISSUE THREAD

**Issue by ktoso** [akka/akka-http#873]
The:

```scala
  def startServer(host: String, port: Int, settings: ServerSettings): Unit = {
```

is missing the simplest overload:

```scala
  def startServer(host: String, port: Int): Unit = ???
```

we should add it.

cc @jlprat

**Comment by ktoso** [comment]
Bumped into this while prepping code for conference demo :-)
I'll see if people like it :-)

**Comment by jlprat** [comment]
You are right, I was having a similar thought some days ago.
I could provide a PR if you like.

Let me know what people say about it :)

**Comment by ktoso** [comment]
Yes please PR if you can. I'll keep feedback coming!

**Comment by jrudolph** [comment]
Fixed by #876.
